agent/src/main/java/com/microsoft/applicationinsights/agent/internal/agent/redis/JedisMethodVisitor.java

```diff
@@ -301,7 +301,7 @@ protected int numberOfArguments() {
         return argumentTypes.length;
     }
 
-    protected Type[] getArgumentTypes() {
+    public Type[] getArgumentTypes() {
```

Comment:
Why did the visibility change here?

Reply:
can't remember right now, so something to do with the asm upgrade, will check

Reply:
![image](https://user-images.githubusercontent.com/21194782/56831642-8a204d80-6861-11e9-9418-5b6d888d8808.png)

Reply:
Will you add `@Override`?